src/pudl/settings.py

```diff
@@ -58,7 +58,7 @@ def validate_partitions(cls, partitions):  # noqa: N805
             if len(partitions_not_working) > 0:
                 raise ValueError(
                     f"'{partitions_not_working}' {name} are not available.")
-            partitions[name] = sorted(set(partition))
+            partitions[name] = tuple(sorted(set(partition)))
```

Comment:
I **think** the previous usage of `tuples` for the partition values were to prevent those values from being altered in a function where (e.g.) the values were being used as a default argument.  I think @bendnorman and I talked about this a little in #1424, and also discussed making the definition of the settings classes a little more straightforward -- embedding these constants in each one of them, rather than attempting to dynamically populate them at the class level.

Reply:
Ah I see that now, I'll remove the `tuple`. And when you talk about embedding constants, is that connected to the issue of creating new generalized `working_partitions` models?

Reply:
IIRC previously Ben was trying to dynamically generate the `working_partitions` attributes in these classes based on information that was stored inside the `WORKING_PARTITIONS` dictionary (which still existed in `pudl.constants` at that point) rather than migrating that information into each of the individual settings class definitions, which I think is what we want to do now -- these classes will be the canonical source of information about what data sources/years/states/tables/etc. can be processed (even if sometimes they obtain that information from elsewhere, as in the case of the working tables), with that information being encapsulated in the working partitions, tables, and validation routines that we define for each of them.